src/main/java/storage/FileManager.java

```diff
@@ -486,6 +517,36 @@ public boolean loadScheduleFromFile(DayList dayList) throws IOException {
         return hasNoErrorsDuringLoad;
     }
 
+    /**
+     * Reloads the data from schedule file to DayList after deletion of workout/plan that is related to the schedule.
+     * @param dayList   The DayList object to store the day schedule in.
```

Comment:
Perhaps there should be an empty space between the method description and `@param`?

Reply:
Thank you for spotting this.